test/video_compositor/Video_compositor_opengl_test.exs

```diff
@@ -0,0 +1,47 @@
+defmodule VideoCompositor.OpenGL.NativeTest do
+  use ExUnit.Case, async: true
+
+  alias Membrane.RawVideo
+  alias Membrane.VideoCompositor.OpenGL.Native
+  alias Membrane.VideoCompositor.Test.Utility
+
+  @tag :tmp_dir
+  test "compose doubled raw video frame on top of each other", %{tmp_dir: tmp_dir} do
+    {in_path, out_path, ref_path} = Utility.prepare_paths("1frame.yuv", "ref-native.yuv", tmp_dir)
```

Comment:
Maybe just generalize this test to take a Native module?

Reply:
I'm not really sure how to do that, since our native modules are not standardized in any way